cmd/wasm-client/main.go

```diff
@@ -596,3 +565,11 @@ func exit(err error) {
 	fmt.Printf("Error running wasm client: %v\n", err)
 	os.Exit(1)
 }
+
+func init() {
+	var err error
+	permsMgr, err = perms.NewManager(true)
```

Comment:
I'm not sure `init()` works correctly with WASM. Can we do that at the beginning of `main()` instead?
I guess we should also test the changes with the demo HTTP page.